<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>maxwell</class>
 <widget class="QWidget" name="maxwell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="config_view">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>681</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>610</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>configuration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="read_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>610</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="directory_edit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>580</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="autocomplete_view">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>681</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>autocomplete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
